pygmt/clib/session.py

```diff
@@ -744,7 +744,7 @@ def put_vector(self, dataset, column, vector):
         first. Use ``family='GMT_IS_DATASET|GMT_VIA_VECTOR'``.
 
         Not at all numpy dtypes are supported, only: float64, float32, int64,
-        int32, uint64, uint32, datetime64 and str_.
+        int32, uint64, uint32, datetime64 and str.
```

Comment:
Here, `str_` means `np.str_`, not the Python `str` type. See https://github.com/GenericMappingTools/pygmt/blob/cc26838d5183e3c71ec1374da5915983ee6ca3ae/pygmt/clib/session.py#L55-L64.

Reply:
Fixed in https://github.com/GenericMappingTools/pygmt/pull/1285/commits/704b6b358a82311242a6986a81e56e2d51495aff

Reply:
Nevermind, not fixed yet.

Reply:
It should be fixed now. If you do not want a raw docstring here, it could work to use a double escape.